typings/index.d.ts

```diff
@@ -1292,19 +1292,16 @@ declare module 'discord.js' {
 		public gateway: string | undefined;
 		public readonly ping: number;
 		public shards: Collection<number, WebSocketShard>;
-		public sessionStartLimit: { total: number; remaining: number; reset_after: number; };
 		public status: Status;
-		public broadcast(packet: any): void;
 	}
 
 	export class WebSocketShard extends EventEmitter {
-		constructor(manager: WebSocketManager, id: number, oldShard?: WebSocketShard);
+		constructor(manager: WebSocketManager, id: number);
 		public id: number;
 		public readonly ping: number;
 		public pings: number[];
 		public status: Status;
 		public manager: WebSocketManager;
-		public send(data: object): void;
```

Comment:
This specific change has me iffy. It makes sense to now show it, since you aren't _meant_ to use it, but for people who use TypeScript and Lavalink, this will cause issues. Should this be removed or be kept, and a warning to be added to the jsdoc?

Reply:
I don't have an issue with making this public in the typings.